bin/add-version.sh

```diff
@@ -49,36 +49,44 @@ do
     case "$1" in
         -f|--file)           PROTOBUF_FILE=$2 && shift 2;;
         -h|--help)           print_usage && exit 1;;
-        -v|--version)        VERSION_TO_ADD=$2 && shift 2;;
+        -t|--timestamp)      TIMESTAMP=$2 && shift 2;;
         --latest)            LATEST=true && shift 1;;
         *)                   break;;
-    esac    
+    esac
 done
 
 if [ "${PROTOBUF_FILE}" == "" ]; then
     print_error_and_exit "Missing protobuf file"
 fi
 
-if [ "${VERSION_TO_ADD}" == "" ]; then
-    print_error_and_exit "Missing version to add"
+if [ "${TIMESTAMP}" == "" ]; then
```

Comment:
would be better if the timestamp was grepped out of the file

Reply:
Oh missunderstood this. I thought we should not use the timestamp of the GAME_MASTER file. But it makes more sense this way, will change that